tests/test_isone.py

```diff
@@ -135,3 +146,57 @@ def test_load_forecast_json_format(self):
         # several items, including date and load
         self.assertIn('BeginDate', data['HourlyLoadForecasts']['HourlyLoadForecast'][0].keys())
         self.assertIn('LoadMw', data['HourlyLoadForecasts']['HourlyLoadForecast'][0].keys())
+
+    def test_lmp_realtime_json_format(self):
+        data = self.c.fetch_data('/fiveminutelmp/current/location/4001.json', self.c.auth)
+
+        # one item, FiveMinLmp
+        self.assertIn('FiveMinLmp', data.keys())
+        self.assertEqual(len(data.keys()), 1)
+
+        # several items, including date and lmp
+        self.assertEqual(len(data['FiveMinLmp']), 1)
+        self.assertIn('BeginDate', data['FiveMinLmp'][0].keys())
+        self.assertIn('LmpTotal', data['FiveMinLmp'][0].keys())
+
+        # values
+        date = dateutil.parser.parse(data['FiveMinLmp'][0]['BeginDate'])
+        now = pytz.utc.localize(datetime.utcnow())
+        self.assertLessEqual(date, now)
+        self.assertGreaterEqual(date, now - timedelta(minutes=6))
+        self.assertEqual(date.minute % 5, 0)
+        self.assertGreater(data['FiveMinLmp'][0]['LmpTotal'], 0)
+
+    def test_lmp_past_json_format(self):
+        data = self.c.fetch_data('/fiveminutelmp/day/20150610/location/4001.json', self.c.auth)
+
+        # one item, FiveMinLmps
+        self.assertIn('FiveMinLmps', data.keys())
+        self.assertEqual(len(data.keys()), 1)
+        self.assertIn('FiveMinLmp', data['FiveMinLmps'].keys())
+        self.assertEqual(len(data['FiveMinLmps'].keys()), 1)
+
+        # 288 elements: every 5 min
+        self.assertEqual(len(data['FiveMinLmps']['FiveMinLmp']), 288)
+
+        # several items, including date and lmp
+        self.assertIn('BeginDate', data['FiveMinLmps']['FiveMinLmp'][0].keys())
+        self.assertIn('LmpTotal', data['FiveMinLmps']['FiveMinLmp'][0].keys())
+
+    def test_get_lmp_latest(self):
+        prices = self.c.get_lmp('NEMASSBOST')
+        self.assertEqual(len(prices), 1)
+        self.assertLess(prices[0]['lmp'], 1500)
+        self.assertGreater(prices[0]['lmp'], -300)
+
+    def test_get_lmp_bad_zone(self):
+        self.assertRaises(ValueError, self.c.get_lmp, 'badzone')
+
+    def test_get_lmp_hist(self):
+        start_at = datetime(
+            2015, 01, 01, 01, 1, 0, 0, tzinfo=pytz.timezone('US/Eastern')).astimezone(pytz.utc)
+        end_at = start_at + timedelta(minutes=55)
+
+        prices = self.c.get_lmp('NEMASSBOST', latest=False, start_at=start_at, end_at=end_at)
+        self.assertEqual(len(prices), 11)
+        self.assertEqual(prices[0]['lmp'], 56.92)
```

Comment:
The price value here is different than in the old test... Which do you think is correct?